### ISSUE TITLE
send via mail

### ISSUE THREAD

**Issue by heylix** [flint-gg/easyshare#18]
you could have people enter an email address and send the screenshots there.

**Comment by T0TProduction** [comment]
Actually, a good idea that I never thought of!

I'll put this on the backburner to be done when either 
- someone has free time to just do this
or
- more people start upvoting this/want this feature

If you have any specific vision of how this should work already ( your specific use case, one email per tweet, or maybe one per day with all tweets? ), feel free to add info in the comments

**Comment by heylix** [comment]
Could be configurable – either a digest of all the pictures once a day or a single email for every picture (no more than 1/minute) or even a button in the UI where you can say "batch send last n pictures". Or an interface where you can select which pictures to (re-)mail to you and/or delete.

**Comment by T0TProduction** [comment]
sounds valid, maybe as an MVP (minimum viable product) having a simple "instead of google photos, email me" setting, which sends emails per tweet we get, is the best way to start.
We would not need to cache images across time spans ( e.g. per day, or maybe even within minutes of multiple tweets ) and then we can enhance the system over time to offer more options

**Comment by n1trux** [comment]
Sounds great, maybe send a link on first activation of that feature to confirm.

And definitely include a link to "unsubscribe" and/or "report this activity" so people can't be spammed with your service.

**Comment by f382** [comment]
Yes, I would also like the e-mail feature. I'm a bit worried about linking google photos because I don't want any third party to have access to my photos

**Comment by T0TProduction** [comment]
> Yes, I would also like the e-mail feature. I'm a bit worried about linking google photos because I don't want any third party to have access to my photos

Just for clarification, you don't want Google Photos to have access to your screenshots, or you don't want this tool to have access to your photos on Google Photos? 
Because the latter is not possible, the Google Photos API only gives access to images we upload ourselves, and not even really for that. It's pretty much one way only.

**Comment by f382** [comment]
Thanks for clarifying that. Then it's less of an issue.
